lib/Component.spec.lua

```diff
@@ -455,31 +458,40 @@ return function()
 
 			local testElement = createElement(TestComponent)
 
-			expect(function()
-				Reconciler.mount(testElement)
-				forceUpdate()
-			end).to.throw()
+			local handle = Reconciler.mount(testElement)
+
+			expect(forceUpdate).to.be.a("function")
```

Comment:
Not a problem, but I'm curious why you added this.

Reply:
My thought process was that this expectation is sort of implicit, and if it ever failed, the error message would be not-so-great. Thinking about it now, I think I'll remove it.